docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -0,0 +1,144 @@
+---
+product: date-pickers
+title: Date and Time Picker - Base concepts
+packageName: '@mui/x-date-pickers'
+githubLabel: 'component: pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.html
+---
+
+# Date and Time Pickers - Base concepts
+
+<p class="description">The Date and Time pickers expose a lot of components to fit your every needs.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic usage
+
+All the components have a `value` / `onChange` API to set and control the values:
+
+{{"demo": "ControlledComponent.js"}}
+
+## Imports
+
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but without the nested imports.
+
+For example, to use the `DatePicker` component, the following three imports are valid:
+
+```tsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { DatePicker } from '@mui/x-date-pickers';
+import { DatePicker } from '@mui/x-date-pickers-pro';
+```
+
+## Other components
+
+### Choose interaction style
+
+Depending on your use case, different interaction styles are preferred.
+
+- For input editing with a popover for mouse interaction, use the _Picker_ components:
```

Comment:
Nitpick: This could potentially use a bit different wording in order to cover mobile dialog behaviour as well.
Also, technically both desktop and mobile shows a dialog (at least depending on the provided `role`) 🙈